gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/vertx/web/handler/impl/AuthenticationFlowHandlerImpl.java

```diff
@@ -43,6 +43,7 @@
     public Handler<RoutingContext> create() {
         List<AuthenticationFlowStep> steps = new LinkedList<>();
         steps.add(new SPNEGOStep(RedirectHandler.create("/login/SSO/SPNEGO"), identityProviderManager));
+        steps.add(new FormTwoStepLoginStep(RedirectHandler.create("/2step/login")));
```

Comment:
Most of the time this feature is called identifier-first authentication, I prefer `/login/identifier` instead of `/2step/login`